preferences/oracle_linux/requirements/requirements.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: requirements
+spec:
+  # https://docs.oracle.com/en/operating-systems/oracle-linux/limits/
+  requirements:
+    cpu:
+      guest: 2
+    memory:
+      guest: 3Gi
```

Comment:
I thought we said 4Gi assuming that the arm64 requirement of 2Gi was per vCPU?

Reply:
IIRC we agreed with 3Gi, but I might be wrong :sweat_smile: 
In https://docs.oracle.com/en/operating-systems/oracle-linux/limits/ it is not very clear if arm requires 2Gi in total or 2Gi per vCPU. In any case, I'm fine with 4Gi just in case.